modules/data-entry/src/main/frontend/src/dataHomepage/LiveTable.jsx

```diff
@@ -268,7 +268,13 @@ function LiveTable(props) {
         entryLabel={entry["jcr:primaryType"] == "cards:Subject" ? entry.type?.label : undefined}
         admin={admin} />
     });
-    return <TableCell key={index} className={classes.tableActions}>{content}</TableCell>;
+    return (
+      <TableCell key={index} className={classes.tableActions}>
+        <Stack direction="row" sx={{justifyContent: "flex-end"}}>
```

Comment:
Not tested, but I think we should remove `className={classes.tableActions}` because:
* the `float: right` should now be covered by  `<Stack direction="row" sx={{justifyContent: "flex-end"}}>`
* the `margin*` settings can be specified on the `Stack` component instead, and by doing that we get rid of the inconsistent tooltip positioning as well
* not sure why we needed to specify the padding for these actions.

Note: `classes.tableActions` is still used elsewhere (in `Filters`) so we can't completely remove the class yet.

```suggestion
      <TableCell key={index}>
        <Stack direction="row" sx={{justifyContent: "flex-end", my: .5}}>
```